### PR TITLE
Added questions on how to add easyfix, mentor

### PR DESCRIPTION
**Pull request by chkumar246**

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -0,0 +1,49 @@
+===================
+RDO Project Easyfix
+===================
+
+It is a collection of easyfix or low hanging fruits issues for RDO Project.
```
Comment (yac) [inline README.rst]: of easy fixes or low hanging fruit issues

File: README.rst [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+
+I am new to RDO Project, How can i contribute?
```
Comment (yac) [inline README.rst]: s/ i / I /

**Comment by javierpena** [comment]
👍

File: README.rst [hunk 3]
```diff
@@ -0,0 +1,49 @@
+===================
+RDO Project Easyfix
```
Comment (yac) [inline README.rst]: I suggest naming README.md which github nicely renders as Markdown including links and... use Markdown ;)

File: README.rst [hunk 4]
```diff
@@ -0,0 +1,49 @@
...
+===================
+
+It is a collection of easyfix or low hanging fruits issues for RDO Project.
+
+I am new to RDO Project, How can i contribute?
+==============================================
+
+* You can check the issues and assign to it yourself.
```
Comment (yac) [inline README.rst]: You can check [Issues](https://github.com/redhat-openstack/easyfix/issues) and assign some to yourself.

File: README.rst [hunk 5]
```diff
@@ -0,0 +1,49 @@
...
+* Read the issue carefully and go through the links attached to issue.
+* If you are not able understand, join the IRC channel #rdo on Freenode server.
+* Ping the Mentor regarding the issue, S/He will help to move forward.
+
+I want to add easyfix, How can i do that?
+=========================================
+* Go to issues and file the issue.
+* The issue should properly define the problem to be solved.
+* Please add tag as project name.
+* It would be great how i can reproduce it.
```
Comment (apevec) [inline README.rst]: * If possible add steps how it can be reproduced.

File: README.rst [hunk 6]
```diff
@@ -0,0 +1,49 @@
...
+* Add links which will help to understand the problem.
+* Adding your IRC link so that people can reach out for help.
+* Once problem is reproduced, where they can submit the changes.
+* Always include project source code link.
+
+Still confused, What should i do?
+=================================
+* Join the `RDO mailing list <https://www.redhat.com/mailman/listinfo/rdo-list>`
```
Comment (amoralej) [inline README.rst]: I'd add a reference to https://www.rdoproject.org/documentation/onboarding/ or https://www.rdoproject.org/documentation/rdo-packaging/

File: README.rst [hunk 7]
```diff
@@ -0,0 +1,49 @@
...
+
+It is a collection of easyfix or low hanging fruits issues for RDO Project.
+
+I am new to RDO Project, How can i contribute?
+==============================================
+
+* You can check the issues and assign to it yourself.
+* Each issue contains mentor IRC nick.
+* Read the issue carefully and go through the links attached to issue.
+* If you are not able understand, join the IRC channel #rdo on Freenode server.
```
Comment (yac) [inline README.rst]: If you aren't able understand, join the IRC channel `#rdo` on Freenode server.

File: README.rst [hunk 8]
```diff
@@ -0,0 +1,49 @@
...
+* Ping the Mentor regarding the issue, S/He will help to move forward.
+
+I want to add easyfix, How can i do that?
+=========================================
+* Go to issues and file the issue.
+* The issue should properly define the problem to be solved.
+* Please add tag as project name.
+* It would be great how i can reproduce it.
+* Add links which will help to understand the problem.
+* Adding your IRC link so that people can reach out for help.
```
Comment (apevec) [inline README.rst]: * Add your IRC nickname...

File: README.rst [hunk 9]
```diff
@@ -0,0 +1,49 @@
...
+RDO Project Easyfix
+===================
+
+It is a collection of easyfix or low hanging fruits issues for RDO Project.
+
+I am new to RDO Project, How can i contribute?
+==============================================
+
+* You can check the issues and assign to it yourself.
+* Each issue contains mentor IRC nick.
```
Comment (yac) [inline README.rst]: Mentor

File: README.rst [hunk 10]
```diff
@@ -0,0 +1,49 @@
...
+* Read the issue carefully and go through the links attached to issue.
+* If you are not able understand, join the IRC channel #rdo on Freenode server.
+* Ping the Mentor regarding the issue, S/He will help to move forward.
+
+I want to add easyfix, How can i do that?
```
Comment (apevec) [inline README.rst]: s/i/I/

File: README.rst [hunk 11]
```diff
@@ -0,0 +1,49 @@
...
+* Go to issues and file the issue.
+* The issue should properly define the problem to be solved.
+* Please add tag as project name.
+* It would be great how i can reproduce it.
+* Add links which will help to understand the problem.
+* Adding your IRC link so that people can reach out for help.
+* Once problem is reproduced, where they can submit the changes.
+* Always include project source code link.
+
+Still confused, What should i do?
```
Comment (apevec) [inline README.rst]: s/i/I/

File: README.rst [hunk 12]
```diff
@@ -0,0 +1,49 @@
...
+It is a collection of easyfix or low hanging fruits issues for RDO Project.
+
+I am new to RDO Project, How can i contribute?
+==============================================
+
+* You can check the issues and assign to it yourself.
+* Each issue contains mentor IRC nick.
+* Read the issue carefully and go through the links attached to issue.
+* If you are not able understand, join the IRC channel #rdo on Freenode server.
+* Ping the Mentor regarding the issue, S/He will help to move forward.
```
Comment (yac) [inline README.rst]: will help you move forward

File: README.rst [hunk 13]
```diff
@@ -0,0 +1,49 @@
...
+
+I want to add easyfix, How can i do that?
+=========================================
+* Go to issues and file the issue.
```
Comment (yac) [inline README.rst]: [Issues](https://github.com/redhat-openstack/easyfix/issues) link for convenience

File: README.rst [hunk 14]
```diff
@@ -0,0 +1,49 @@
...
+* Add links which will help to understand the problem.
+* Adding your IRC link so that people can reach out for help.
+* Once problem is reproduced, where they can submit the changes.
+* Always include project source code link.
+
+Still confused, What should i do?
+=================================
+* Join the `RDO mailing list <https://www.redhat.com/mailman/listinfo/rdo-list>`
+* Introduce yourself.
+* OR you can join #rdo IRC channel on Freenode server and Say hello.
```
Comment (apevec) [inline README.rst]: s/server//
s/Say/say/

File: README.rst [hunk 15]
```diff
@@ -0,0 +1,49 @@
...
+* We are always there, Happy to Help.
```
Comment (apevec) [inline README.rst]: s/Happy/happy/ s/Help/help/

File: README.rst [hunk 16]
```diff
@@ -0,0 +1,49 @@
...
+
+I want to become Mentor, What should I do?
+==========================================
+* Add your name to Mentor list with IRC nick.
+* Follow I want to add easyfix, How can i do that?.
```
Comment (apevec) [inline README.rst]: s/i/I/

File: README.rst [hunk 17]
```diff
@@ -0,0 +1,49 @@
...
+* Each issue contains mentor IRC nick.
+* Read the issue carefully and go through the links attached to issue.
+* If you are not able understand, join the IRC channel #rdo on Freenode server.
+* Ping the Mentor regarding the issue, S/He will help to move forward.
+
+I want to add easyfix, How can i do that?
+=========================================
+* Go to issues and file the issue.
+* The issue should properly define the problem to be solved.
+* Please add tag as project name.
```
Comment (yac) [inline README.rst]: Please add `[PROJECT] ` prefix to Issue name. ?

Alternatively, labels can be used, but I guess this is clearer on the first sight.

**Review by apevec — CHANGES_REQUESTED** [review]

File: README.rst (hunk 5 above)
Comment (yac) [inline README.rst]: Describe howto reproduce the issue.

File: README.rst [hunk 18]
```diff
@@ -0,0 +1,49 @@
...
+* It would be great how i can reproduce it.
+* Add links which will help to understand the problem.
```
Comment (yac) [inline README.rst]: Add links to help understand the problem.

File: README.rst (hunk 8 above)
Comment (yac) [inline README.rst]: s/Adding/Add/ for consistency with above point

File: README.rst [hunk 19]
```diff
@@ -0,0 +1,49 @@
...
+* Adding your IRC link so that people can reach out for help.
+* Once problem is reproduced, where they can submit the changes.
```
Comment (yac) [inline README.rst]: Describe where/how to submit required changes.

File: README.rst (hunk 11 above)
Comment (yac) [inline README.rst]: s/ i / I /

File: README.rst (hunk 6 above)
Comment (yac) [inline README.rst]: Join the [RDO mailing list](https://www.redhat.com/mailman/listinfo/rdo-list).

File: README.rst (hunk 16 above)
Comment (yac) [inline README.rst]: s/ i / I /

File: README.rst [hunk 20]
```diff
@@ -0,0 +1,49 @@
...
+* OR you can join #rdo IRC channel on Freenode server and Say hello.
+* We are always there, Happy to Help.
+
+I want to become Mentor, What should I do?
+==========================================
+* Add your name to Mentor list with IRC nick.
+* Follow I want to add easyfix, How can i do that?.
+
+
+Mentors
```
Comment (yac) [inline README.rst]: Mentors (IRC nicks)

File: README.rst [hunk 21]
```diff
@@ -0,0 +1,49 @@
...
+=======
+* Alfredo Moralejo (amoralej)
+* Alan Pevec (apevec)
+* Chandan Kumar (chandankumar)
+* David Moreau Simard (dmsimard)
+* Haikel Guemar (number80)
+* Jakub Ruzicka (jruzicka)
```
Comment (yac) [inline README.rst]: * Jakub Ruzicka (`jruzicka`)
(distinguish `nick` with monospace)

**Review by yac — DISMISSED** [review]
Got some suggestions ;)

**Review by apevec — APPROVED** [review]

**Pull request merged**